pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -523,7 +526,10 @@ public boolean jdbcCompliant() {
    * @return Properties with elements added from the url
    */
   public static Properties parseURL(String url, Properties defaults) {
-    Properties urlProps = new Properties(defaults);
+    Properties urlProps = new Properties();
+    if (defaults != null) {
+      urlProps.putAll(defaults);
```

Comment:
I've created "Properties, defaults, PR 900, etc" discussion on the pgjdbc-jdbc mailing list.
I would appreciate @whitingjr and/or @alexismeneses to step in as well.

Just in case: `urlProps.putAll(defaults);` will use `defaults.entrySet()` and it will ignore "parent properties" of a default object.

It happens so that this particular `Properties defaults` is initialized with "parent Properties" object (see https://github.com/pgjdbc/pgjdbc/blob/ede2da1898aee4e503e1041c133dbf95322fa4a8/pgjdbc/src/main/java/org/postgresql/Driver.java#L220 ), so I expect this change will break `getDefaultProperties();`.